### DISCUSSION TITLE
专业版付款失败

### DISCUSSION THREAD

**Discussion by zangrc** [getlantern/lantern-forum-cn#369] [Q&A]
![image](https://github.com/getlantern/lantern-forum-cn/assets/15864353/fec2e068-025b-41cf-ab1b-1437c6566b90)
蓝灯专业版支付的时候提示付款时发生错误，换其他银行卡也不行，大家是怎么支付成功的

**Comment by MaxMKMax** [comment]
@zangrc 您好. 这是我们的另一个购买渠道. 可以用支付宝付费的. 谢谢您支持蓝灯!
https://buy.stripe.com/eVacPZe3A5cp1oc4gO

**Reply by zangrc** [reply]
请问有1年的购买渠道吗？

**Reply by MaxMKMax** [reply]
@zangrc 有的. 您可以选一年. 您在链接里找不到一年的选项吗? |

我这里可以帮您什么忙呢?

**Reply by zangrc** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/15864353/22e68d2a-470b-43f8-bde5-155fdcbe44d6)
打开链接后，直接显示2年的购买渠道，没有找到切换选项

**Comment by MaxMKMax** [comment]
@zangrc 请问您是通过哪一个地方或页面购买的?
